Britannia/common/named_colors/default_colors.txt

```diff
@@ -1,22 +1,21 @@
 colors = {
```

Comment:
Rather than overriding the vanilla file, it's probably better just to add the new colours in a separate file

Reply:
Fixed